src/utils/Constants.js

```diff
@@ -1,11 +1,91 @@
+/* Cache constants */
+const AUTOCOMPLETE_CACHE_OPTIONS = {
+  max: 10000,
+  maxAge: 1000 * 60 * 60 * 24 * 5,
+};
+const ROUTES_CALC_CACHE_OPTIONS = {
+  max: 1000, // max 1000 routes in storage
+  maxAge: 1000 * 60 * 15, // max age 15 minutes
+};
+const QUERY_PREDICTIONS_CACHE_OPTIONS = {
+  max: 10000, // Maximum size of cache
+  maxAge: 1000 * 60 * 60 * 24 * 5, // Maximum age in milliseconds
+};
+
+/* Count constants */
+const MIN_FUZZ_RATIO = 75;
+
+/* Delay Buffer constants */
+// buffer to account for routes in past 20 minutes with delays
+const FIRST_DELAY_BUFFER_IN_MINUTES = 20;
+// additional buffer to account for time needed to walk from current location to bus stop
+const SECOND_DELAY_BUFFER_IN_MINUTES = 40;
+
+/* Distance & Speed constants */
+// > 3.0 suggests getting off bus earlier and walk half a mile instead of waiting longer
+const MAX_WALK_DIST_PER_LEG = 2000;
+const AUTOCOMPLETE_RADIUS = 24140;
+const WALK_SPEED = 3.0;
+
+/* Degrees Precision constants */
+const DEG_EQ_PRECISION = 5; // 5 degrees of precision is about a 1.1 meters, is a stop
+const DEG_EXACT_PRECISION = 6; // 6 degrees of precision is about a 111 mm, is exact point
+const DEG_KM_PRECISION = 2; // 3 degrees of precision is about 1 km, stop barely walkable
+const DEG_MAX_PRECISION = 6;
+const DEG_MIN_PRECISION = 1;
+const DEG_NEARBY_PRECISION = 4; // 4 degrees of precision is about 11 meters, stop nearby
+const DEG_WALK_PRECISION = 3; // 3 degrees of precision is about 111 meters, stop walkable
```

Comment:
I think we should actually not alphabetize and have these in order, it'd be clearer for the future

Reply:
these don't really seem to reference the same value, how would you order them?

Reply:
oh i meant in terms of like 1-6 in terms of value, but I don't have a strong preference!

Reply:
oh duh LOL me neither